src/tls.c

```diff
@@ -11529,10 +11534,23 @@ int TLSX_PopulateExtensions(WOLFSSL* ssl, byte isServer)
 
                 /* Pre-shared key is mandatory extension for resumption. */
                 ret = TLSX_PreSharedKey_Use(ssl, sess->ticket, sess->ticketLen,
-                                            milli, ssl->specs.mac_algorithm,
-                                            ssl->options.cipherSuite0,
-                                            ssl->options.cipherSuite, 1,
-                                            NULL);
+                    milli, ssl->specs.mac_algorithm, ssl->options.cipherSuite0,
+                    ssl->options.cipherSuite, 1, NULL);
+            #else
+                seen = (sword64)sess->ticketSeen * 1000 + sess->ticketSeenMilli;
+                now = TimeNowInMilliseconds();
+                if (now < seen)
+                    milli = (0xFFFFFFFFFFFFFFFFU - seen) + 1 + now;
```

Comment:
Is this still correct? this was for a wrap around of 32 bit unsigned value (which was after 1.6 months).
We are now dealing with a signed value, which in practice will wrap around after 292471208,677536 years.
Theoretically this still could happen if for some reason your if TimeNowInMilliseconds is offsetted with some huge number for some reason. But then this 0xFFFFFFFFFFFFFFFFU seems wrong to me: the U is for unsigned. Not sure how to wrap around for a signed value. A sword 64 time should never overflow in my opinion...

Reply:
Removed.